model-optimizer/mo/utils/check_ie_bindings.py

```diff
@@ -19,15 +17,22 @@
     execution_type = "install_prerequisites.{}".format("bat" if platform.system() == "Windows" else "sh")
 
 import mo.utils.version as v
-import telemetry.telemetry as tm
+try:
+    import openvino_telemetry as tm  # pylint: disable=import-error,no-name-in-module
+except ImportError:
+    import mo.utils.telemetry_stub as tm
 from mo.utils.error import classify_error_type
 
 
 def send_telemetry(mo_version: str, message: str, event_type: str):
-    t = tm.Telemetry(app_name='Model Optimizer', app_version=mo_version)
-    t.start_session()
+    t = tm.Telemetry(app_name='Version Checker', app_version=mo_version)
+    # do not trigger new session if we are executing from the check from within the MO because it is actually not model
+    # conversion run which we want to send
+    if execution_type != 'mo':
+        t.start_session(execution_type)
```

Comment:
don't we want to send statistics from C-code, for example, `offline_transformations` because it is more complex to hack?

Reply:
It is a valid concern. But the issue is that anyone can write a 5 line script in Python and send a lot of data to the GA. The GA backend itself has a mechanism preventing from such a DOS attack, so we fully rely on it.